pkgs/applications/editors/vim/plugins/deprecated.json

```diff
@@ -19,6 +19,9 @@
         "date": "2022-10-20",
         "new": "neodev-nvim"
     },
+    "nvchad-extensions": {
+        "date": "2023-08-19"
```

Comment:
might be worth mentioning nvchad-ui here since that seems to be the successor of the repo: https://github.com/NvChad/NvChad/issues/2295